change/@azure-msal-node-2020-11-25-19-29-14-device-code-timeout.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
```

Comment:
Looks like msal-node changes are only package-lock? This should be type none in that case

Reply:
Change this to a `none`.